crates/lib/src/qpu/quilc/mod.rs

```diff
@@ -195,7 +195,7 @@ mod tests {
         let output = compile_program(
             "MEASURE 0",
             TargetDevice::try_from(qvm_isa()).expect("Couldn't build target device from ISA"),
-            &Configuration::default(),
+            &QcsClient::with_config(ClientConfiguration::default()),
```

Comment:
Is this different from `&QcsClient::default()`?

Reply:
Also, IIRC, a default `ClientConfiguration` doesn't have any credentials associated with it.

Reply:
There is no `QcsClient::default()`, and it's not much of a concern that there are no credentials since this test doesn't need them. But I'll replace this with `QcsClient::load()` anyway.